selenium_tests/lti_emailer/emailer_main_page_tests.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+
+from selenium_tests.lti_emailer.emailer_base_test_case import EmailerBaseTestCase
+from selenium_tests.lti_emailer.page_objects.emailer_mainpage_page_object import EmailerMainPage
+
+
+class EmailerMainPageTests(EmailerBaseTestCase):
+
+    def test_tool_is_loaded_on_page(self):
+        """Check that page is loaded by checking against site title"""
+        driver = self.driver
+        page = EmailerMainPage(driver)  # instantiate
+        page.is_loaded()
```

Comment:
Good point on the assert.  Josie had a similar comment on the existence of this file.  This was just a is_loaded test (basic check to make sure that tool loaded on the page) and not anything else (useful only if the title check does not pass). The key test for this sprint is the checks against  tool_access_permission, so I'm going to remove this file altogether as it is confusing to have it in there to just test is_loaded when there isn't anything I am testing in particular right now for the mainpage.